### ISSUE TITLE
unable to list the resources of eks by assuming role (from kubeconfig)

### ISSUE THREAD

**Issue by phanisowjanyavutukuri-mn** [kontena/lens#4581]
Hi,

I have access to AWS account A and AWS account B for a user which exists in Account A, so i have to assume the role and get the resources, while doing it from cmd/powershell I am able to get the eks resources using the same kubeconfig but in lens iam unable to
get the resources by showing the below error:
**Cluster <account-no>:cluster/<cluster-name> does not have permissions to list namespaces. Please add the namespaces you have access to.**

**Comment by jim-docker** [comment]
Does the role have permissions to get namespaces? Does `kubectl get namespaces` work? Does it help if you add accessible namespaces on the cluster settings page?

**Comment by phanisowjanyavutukuri-mn** [comment]
yes, with the same role am able to access all the namespaces via powershell or cmd, but not via lens

**Comment by jim-docker** [comment]
The cluster in Lens corresponds to a context in your kubeconfig. Lens respects the RBAC rules set for the user associated with that context. What does that context look like? Can you share (some of) your kubeconfig? Does `kubectl get namespaces` work in the Lens terminal (when connected to that cluster)?
